win32ss/drivers/videoprt/int10.c

```diff
@@ -23,11 +23,61 @@
 
 #include <ndk/kefuncs.h>
 #include <ndk/halfuncs.h>
+#include <ndk/mmfuncs.h>
 
 #define NDEBUG
 #include <debug.h>
 
 /* PRIVATE FUNCTIONS **********************************************************/
+#define IsLowV86Mem(_Seg, _Off) ((((_Seg) << 4) + (_Off)) < (0xa0000))
+
+/* Those two functions below are there so that CSRSS can't access low mem.
+ * Expecially, MAKE IT CRASH ON NULL ACCESS */
```

Comment:
Well I thought having this access in CSRSS was done also like that in order for the console (that runs in CSRSS context) in fullscreen mode (OK that's what happens on Windows, not on ROS... yet??) to access the VGA video buffer and stuff.

Reply:
The frame buffer is at 0xA0000, above the protected range.

Reply:
OK this sounds good.